### ISSUE TITLE
Sequence to One RNN for voiceprint recognition

### ISSUE THREAD

**Issue by ForeverGaara** [Element-Research/rnn#251]
hello @nicholas-leonard 

I am very interested in **Sequence to One RNN**, woud it be possible to apply RNN on **voiceprint recognition/speaker recognition**, and what kind structure of RNN can be used to do this task.  looking forward your reply, thank you.

**Comment by nicholas-leonard** [comment]
Yes. You can checkout our [sequence-to-one.lua](https://github.com/Element-Research/rnn/blob/master/examples/sequence-to-one.lua) script. The key is to use a `SelectTable(-1)` after your `Sequencer(RNN)`.

**Comment by Trojahn** [comment]
Hi @nicholas-leonard!

How can i change the example script to:
1 - Support 1D vector values instead of simple numbers? (Each sample have the same size)
2 - Support sequences of variable size? (Even considering batch_size = 1)

My aim is to classify if a given sequence (formed by a bunch of 1D vectors values) is either of a given class or not (binary classifier). 

Thank you!
